dist/operators/s2r.d.ts

```diff
@@ -42,5 +42,10 @@ export declare class CSPARQLWindow {
     add(e: Quad, timestamp: number): void;
     compute_report(w: WindowInstance, content: QuadContainer, timestamp: number): boolean;
     scope(t_e: number): void;
-    subscribe(output: 'RStream' | 'IStream' | 'DStream', call_back: (data: QuadContainer) => void): void;
+    subscribe(output: 'RStream' | 'IStream' | 'DStream', call_back: (data: container_with_bounds) => void): void;
```

Comment:
I would suggest to just pass a metadata object with the QuadContainer in stead, so it is more flexible. Also I dont like the name ContainerWithBounds, when picking a name for a class/function/variable it should be clear from the name what it describes/does/contains...